File: contacting-conf-web/src/main/scala/nl/ing/api/contacting/conf/repository/model/SurveyDetails.scala
package nl.ing.api.contacting.conf.repository.model

import nl.ing.api.contacting.conf.repository.SurveyOrgRepository.SurveyOrgDetails
import nl.ing.api.contacting.conf.repository.SurveyTaskQRepository.SurveyTaskQDetails

/**
 * @author Ayush Mittal
 */
case class SurveyDetails(setting: SurveySetting, phNumFormat: Seq[SurveyPhoneNumberFormat], taskQMapping: Seq[SurveyTaskQDetails], orgMapping: Seq[SurveyOrgDetails])

File: contacting-conf-web/src/main/java/nl/ing/api/contacting/conf/domain/model/surveysetting/SurveyDetails.java
package nl.ing.api.contacting.conf.domain.model.surveysetting;

import nl.ing.api.contacting.conf.domain.entity.SurveyPhoneNumberFormatEntity;
import nl.ing.api.contacting.conf.domain.entity.SurveySettingsEntity;

import java.util.List;

public record SurveyDetails(
        SurveySettingsEntity setting,
        List<SurveyPhoneNumberFormatEntity> phNumFormat,
        List<SurveyTaskQMappingWithName> taskQMapping,
        List<SurveyOrgDetails> orgMapping
) {}
